src/sentry/identity/oauth2.py

```diff
@@ -137,15 +137,42 @@ def handle_refresh_error(self, req, payload):
         formatted_error = u'HTTP {} ({}): {}'.format(req.status_code, error_name, error_description)
 
         if req.status_code == 401:
+            self.logger.info(
+                'identity.oauth.refresh.identity-not-valid-error',
+                extra={
+                    'error_name': error_name,
+                    'error_status_code': req.status_code,
+                    'error_description': error_description,
+                    'provider_key': self.key,
```

Comment:
Is there any data we could use to tie this back to the user/organization that it occurred within?